docs/redoc/v0.10.0/openapi.json

```diff
@@ -3229,7 +3229,7 @@
         }
       },
       "NamedVectorStruct": {
-        "description": "Vector data separator for named and unnamed modes Unanmed mode:\n\n{ \"vector\": [1.0, 2.0, 3.0] }\n\nor named mode:\n\n{ \"vector\": { \"vector\": [1.0, 2.0, 3.0], \"name\": \"image-embeddings\" } }",
+        "description": "Vector data separator for named and unnamed modes Unnamed mode:\n\n{ \"vector\": [1.0, 2.0, 3.0] }\n\nor named mode:\n\n{ \"vector\": { \"vector\": [1.0, 2.0, 3.0], \"name\": \"image-embeddings\" } }",
```

Comment:
This probably comes from here: https://github.com/qdrant/qdrant-web-ui/blob/5bf5f68291fa5a69b1021e7d57cd41f792662b47/public/openapi.json#L4171
And thus should probably be dropped and fixed there...

Reply:
It is sourced from here, and already fixed:
https://github.com/qdrant/qdrant/blob/a6164ccda89b00eb361686ed40b90da9fbb1b2c3/docs/redoc/master/openapi.json#L5700-L5702